pkg/pillar/cmd/zedkube/zedkube.go

```diff
@@ -38,6 +38,12 @@ var (
 	log    *base.LogObject
 )
 
+// GetKubePodsError is used to check and handle get kube pods error
+type GetKubePodsError struct {
+	getKubePodsErrorTime    *time.Time
```

Comment:
Why a pointer type?
Elsewhere we check and set for zero using func (Time) IsZero.

Reply:
Sure. Updated.

Reply:
Thanks